src/main/java/org/hyperledger/aries/api/endorser/EndorseTransactionEvent.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/acapy-java-client
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.aries.api.endorser;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.hyperledger.aries.api.serializer.JsonObjectDeserializer;
+import org.hyperledger.aries.api.serializer.JsonObjectSerializer;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Webhook event: issuer_cred_rev
+ */
+@Data @NoArgsConstructor @AllArgsConstructor
+public class EndorseTransactionEvent {
+    private TransactionType type;
```

Comment:
I have tried this locally and the type looks different, I get the following in the webhook event:

```
    "_type": "http://didcomm.org/sign-attachment/%VER/signature-request",
```

The different types as defined in the enum are in the swagger api, but in the rest response they are either null or not set at all.